.github/workflows/build_docs.yml

```diff
@@ -0,0 +1,114 @@
+name: Build Documentation
+
+# Build documentation on:
+# - pushes to master and uploads to latest folder
+# - Released and uploads to folder that matches release tag
+on:
+  push:
+    branches:
```

Comment:
is it worth setting this to `pr` so that the pipeline can run for the PR and then when it's ok and working re-set it to `master`?